### DISCUSSION TITLE
Can the front-end style of perseus be used with tailwind?

### DISCUSSION THREAD

**Discussion by Ahidada** [seanpmyers/loremaster#450] [Ideas]
Can the front-end style of perseus be used with tailwind?
I have seen examples of using perseus + tailwind, are there some features of tailwind that cannot be achieved?

**Comment by seanpmyers** [comment]
I'm not sure.

I haven't really used tailwind myself.

From my understanding though the styling (CSS) probably works fine with perseus/sycamore if you just include tailwind using the content delivery network (CDN) [tailwind cdn page](https://tailwindcss.com/docs/installation/play-cdn). 

The JavaScript features of tailwind might have some issues though, but I'm pretty sure you can still get it working with some effort. This documentation from perseus could help with that [perseus JavaScript interop](https://framesurge.sh/perseus/en-US/docs/0.4.x/fundamentals/js-interop).

Personally I prefer to use Rust with sycamore/perseus for as much of the client side logic as possible. I also don't really want to rely on external frameworks for my long term project, and I like to avoid JavaScript as much as possible 😃 .

**Comment by Ahidada** [comment]
It does feel uncomfortable to see an extra node_modules in the rust project.

**Comment by seanpmyers** [comment]
Yeah lol
